src/targets.rs

```diff
@@ -1441,12 +1455,13 @@ mod tests {
             "x86_64-unknown-freebsd",
             "x86_64-unknown-haiku",
             "x86_64-unknown-hermit",
-            "x86_64-unknown-hermit-kernel",
```

Comment:
For compatibility, we should ideally continue to recognize old triples and map them to their corresponding new triples. Would it be feasible to teach the parser to recognize this triple and map it to x86_64-unknown-none-hermitkernel, which I assume is the replacement?

Reply:
Ahh yes, I can do that. Rustc has removed/changed quite a few over time, how far back should I support?

Reply:
I don't think we need to add all the old ones, unless someone files an issue specifically asking for them. It's just nice to recognize the ones that target-lexicon itself has previously recognized, so that we don't break existing users.

Exceptions to this would be if a triple is actively incorrect or ambiguous, or if there's a triple we have reason to believe no one is using.